assets/aml-benchmark/tests/test_dataset_preprocessor.py

```diff
@@ -90,6 +90,10 @@ class TestDatasetPreprocessorComponent:
                 """ {"premise":{{premise}}, "hypothesis":{{hypothesis}},"label":{{label|string}}} """,
                 None, '{"column_name":"label", "0":"NEUTRAL", "1":"ENTAILMENT", "2":"CONTRADICTION"}'
             ),
+            (
+                "hellaswag_hf", Constants.PROCESS_SAMPLE_EXAMPLES_INPUT_FILE, None,
+                os.path.join(Constants.CUSTOM_PREPROCESSOR_SCRIPT_PATH, "hellaswag_hf.py"), None
+            ),

```

Comment:
There is no need to test the same thing via script as well as AML run. We should minimize the number of tests that we submit to AML workspace. In this case, we should keep the script test, and eliminate the component test because both will test the same thing.

Reply:
I have seen in some cases, the code works on local via script but fails via component, sometimes related to json conversion of arguments. That is the reason I keep on adding the cases just to make sure that this will work. Although I do test them on local, so I will remove it as I have already confirmed that it works.